src/estimation/src/BerdyHelper.cpp

```diff
@@ -638,6 +652,13 @@ IndexRange BerdyHelper::getRangeLinkSensorVariable(const BerdySensorTypes sensor
         }
     }
 
+
+    if (sensorType == COM_ACCELEROMETER_SENSOR && m_options.berdyVariant == BERDY_FLOATING_BASE)
```

Comment:
You could move this to a separate function, maybe? 

Since it is not associated with any link, rather a Frame.

Reply:
Updated in https://github.com/robotology/idyntree/pull/571/commits/2f12467492a2ba8216d1ced0ad4ba8531e7f49a3